app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/account/components/draft_list_item.scala.html

```diff
@@ -0,0 +1,67 @@
+@*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import cats.syntax.order._
+@import java.time.LocalDate
+@import uk.gov.hmrc.cgtpropertydisposalsfrontend.controllers
+@import uk.gov.hmrc.cgtpropertydisposalsfrontend.models.finance.AmountInPence
+@import uk.gov.hmrc.cgtpropertydisposalsfrontend.models.returns.ReturnSummary
+@import uk.gov.hmrc.cgtpropertydisposalsfrontend.models.LocalDateUtils
+@import uk.gov.hmrc.cgtpropertydisposalsfrontend.models.returns.DraftReturn
+
+@this()
+
+@(draftReturn: DraftReturn)(implicit messages: Messages)
+
+@key = @{draftReturn.id}
+@completionDate =  @{draftReturn.triageAnswers.fold(
+  _ => None,
+  c => Some(c.completionDate.value)
+)}
+@dueDate = @{completionDate.map(d => d.plusDays(30L))}
+
+
+<li class="single-return form-bundle">
+  <div class="grid-row">
+    <div class="column-three-quarters submission-summary">
+      <span id="sentDate-@key" class="caption">
+        @messages("drafts.list.lastUpdated", LocalDateUtils.govShortDisplayFormat(draftReturn.lastUpdatedDate))
+      </span>
+      @if(completionDate.nonEmpty) {
+        <h4>
+          @if(draftReturn.propertyAddress.nonEmpty) {
+            @draftReturn.propertyAddress.map(a => s"${a.line1}, ${a.postcode.value}")
+          } else {
+            @completionDate.map(f => LocalDateUtils.govShortDisplayFormat(f))
+          }
+        </h4>
+      }
+    </div>
+    <div class="column-one-quarter financial-summary">
+      @if(completionDate.nonEmpty) {
```

Comment:
and you can do `dueDate.map` here - that should be what decides whether or not you show anything. The due date is only defined if the completion date is defined, so that condition would be implicitly be checked when mapping on the due date.

Reply:
Done